seed_data/bonfires.json

```diff
@@ -493,15 +493,15 @@
     "name": "Pairwise",
     "difficulty": "2.16",
     "description": [
-      "Return the sum of all the indexes of elements that can be added to another element of the first array and equal the target provided in the second argument",
+      "Return the sum of all indices of elements of 'arr' that can be paired with one other element to form a sum that equals the value in the second argument 'arg'. If multiple sums are possible, return the smallest sum. Once an element has been used, it cannot be reused to pair with another.",
```

Comment:
I think this might be a clearer problem statement.

Instead of using FIFO, I just said that if multiple answers are possible, return the smallest sum, which implies FIFO. Also, the "no-reuse" rule was moved out of the further instructions drop-down.